File: api/Entities/HR/ChecklistHRItem.cs
namespace api.Entities.HR
{
    public class ChecklistHRItem: BaseEntity
    {
        public int ChecklistHRId { get; set; }
        public int SrNo {get; set;}
        public string Parameter {get; set;}
        public bool Accepts {get; set;}=false;
        public string Response {get; set;}
        public string Exceptions {get; set;}
        public bool MandatoryTrue {get; set;}
    }
}

File: client/src/app/_models/hr/checklistHRItem.ts
export interface IChecklistHRItem
{
     id: number;
     checklistHRId: number;
     srNo: number;
     parameter: string;
     accepts: boolean;
     response: string;
     exceptions: string;
     mandatoryTrue: boolean;
}

export class ChecklistHRItem implements IChecklistHRItem
{
     id= 0;
     checklistHRId= 0;
     srNo= 0;
     parameter= '';
     accepts = false;
     response= '';
     exceptions= '';
     mandatoryTrue = false;
}